src/cocotb/share/lib/vpi/VpiImpl.cpp

```diff
@@ -317,7 +317,7 @@ GpiObjHdl *VpiImpl::native_check_create(const std::string &name,
 
     new_hdl = vpi_handle_by_name(const_cast<char *>(fq_name.c_str()), NULL);
 
-#ifdef ICARUS
+#if defined(ICARUS) || defined(VERILATOR)
```

Comment:
If the scope of this PR is just the split, this should not be included. Move this to a follow-on that addresses any changes for generate scope support.

Reply:
oops yeah this wasn't supposed to go in, was experimenting with https://github.com/verilator/verilator/pull/4913